src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -200,6 +202,7 @@ class WorkspaceInitialPage extends React.Component {
                                 iconRight={item.iconRight}
                                 onPress={() => item.action()}
                                 shouldShowRightIcon
+                                fallbackIcon={Expensicons.FallbackWorkspaceAvatar}
```

Comment:
We don't need to pass it here. If so why?

Reply:
Yes, this is unnecessary, now its removed